bundles/AdminBundle/Controller/Admin/Document/DocumentControllerBase.php

```diff
@@ -214,8 +214,8 @@ protected function getFromSession($doc)
             // see also PageController::clearEditableDataAction() | this is necessary to reset all fields and to get rid of
             // outdated and unused data elements in this document (eg. entries of area-blocks)
 
-            if ($sessionDocument = Model\Document\Service::getElementFromSession('document', $doc->getId())
-                            && $documentForSave = Model\Document\Service::getElementFromSession('document', $doc->getId(), '_useForSave')) {
+            if (($sessionDocument = Model\Document\Service::getElementFromSession('document', $doc->getId())) &&
+                ($documentForSave = Model\Document\Service::getElementFromSession('document', $doc->getId(), '_useForSave'))) {
```

Comment:
There might be a bug here at the moment. PHPStan means the variable is a boolean. First the && is executed and then the =. But in right side there is also a =. I don't know how this is handled. With braces it should be safe.